source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -7103,6 +7108,13 @@ export class AdaptiveCard extends ContainerWithActions {
             renderedElement.style.removeProperty("minHeight");
         }
 
+        // if rtl is set to dir at the parent view of carousel, carousel's view will get disappeared
```

Comment:
is this an issue in `swiper`? I'm interested to find out what's happening...

Reply:
I believe so 
![Screen Shot 2021-11-10 at 11 02 38 AM](https://user-images.githubusercontent.com/4112696/141176744-0ca3bb6e-d825-4057-8c6e-0241dd4f7c11.png)
note that the position of actions is correctly shown with rtl set.

Reply:
i found the solution to this.